setup/README.md

```diff
@@ -0,0 +1,36 @@
+# Setup
+
+This is a supporting custom Python module dedicated to the automated installation of the Talawa API in a device. 
+
+## Contents
+
+It contains the following files and subdirectories:
+
+```sh
+
+setup
+├── __init__.py
+├── installation.py
+├── post_installation.py
+├── utils.py
+└── markdown
+    ├── About.md
+    └── Input.md
+
+```
+
+A brief explanation of each of the contents of this file is given below :
+
+- installation.py : This file contails all the functions that are necessary to execute the installation of the Talawa API. 
```

Comment:
Can you please add a brief description of each function especially user_input which is responsible for env creation

Reply:
Description for each and every function is present within the function body, following the [Google Python Style Guide](https://google.github.io/styleguide/pyguide.html). you can read more about the ```user_input``` function from [```installation.py```](https://github.com/kailash360/talawa-api/blob/kailash360/setup/installation.py#L37)

Reply:
You are following the google python style guide in the python file. I am asking to change documentation (.md). Please add a function description here too (brief one).

Reply:
I have added the function descriptions now.